server/routes/v1/model.spec.ts

```diff
@@ -73,6 +73,14 @@ describe('test model routes', () => {
     expect(res.body.version).toBe('1')
   })
 
+  test('that we can delete a specific version for a model', async () => {
+    const resDelete = await authenticatedDeleteRequest(`/api/v1/model/${modelUuid}`)
+    validateTestRequest(resDelete)
```

Comment:
The delete API should return the model UUID, so I would also check for that in the response